.travis.yml

```diff
@@ -8,9 +8,9 @@ cache:
 language: scala
 
 scala:
-  - 2.10.6
```

Comment:
We can't continue Scala 2.10 support if use `scala-collection-compat` https://github.com/scala/scala-collection-compat/issues/1